packages/shared/README.md

```diff
@@ -2,14 +2,13 @@
 
 Here we store data structures that we need to use from both the Client and the Server.
 
-## How to generate FlatBuffers code
+## How to generate ProtoBuf code
 
-The FlatBuffers classes are needed here because the Client need to use them to serialize
-and the server to deserialize them.
+You will need protoc installed (for example, on macOS you can install it with `brew install protobuf`).
```

Comment:
As I understand only the developer of this package needs to install `protoc` to generate the TypsScript files in the `gen` folder.  As this folder is committed to github, this is only needed when these files need to be generated again. Right? (or wrong?)

Reply:
Would be nice to give a short explanation for those of us to which protobuf is completely new :-).